server/services/scheduler/scheduler_test.go

```diff
@@ -35,8 +35,8 @@ func TestCreateTask(t *testing.T) {
 
 func TestCreateRecurringTask(t *testing.T) {
 	taskName := "Test Recurring Task"
-	taskTime := time.Millisecond * 200
-	taskWait := time.Millisecond * 100
+	taskTime := time.Millisecond * 500
+	taskWait := time.Millisecond * 200
```

Comment:
This test was intermittently failing on macOS. Increasing the duration seems to make it stable, but we can continue to monitor to see if it needs to be refactored more.